src/main/java/seedu/address/logic/commands/ToggleViewCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.Model;
+
+/**
+ * Toggles application between showing and hiding students' parent details
+ */
+public class ToggleViewCommand extends Command {
+    public static final String COMMAND_WORD = "toggleView";
+
+    public static final String MESSAGE_SUCCESS_HIDE = "Hiding parent details";
+
+    public static final String MESSAGE_SUCCESS_SHOW = "Showing parent details";
```

Comment:
Perhaps the success messages can be in this tense for standardisation with other commands:
"Parent details are hidden"
and
"Parent details are shown"

Reply:
Gotcha, thanks!